open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerOperationalServices.java

```diff
@@ -70,56 +69,43 @@
      * start and returns the error.  Otherwise it continues through the list, returning the successful
      * start up messages.
      *
-     * @param userId  user that is issuing the request
-     * @param serverNames  list of server names
+     * @param userId      user that is issuing the request
+     * @param serverNames list of server names
      * @return success message or
      * OMAGNotAuthorizedException the supplied userId is not authorized to issue this command or
      * OMAGInvalidParameterException the server name is invalid or
      * OMAGConfigurationErrorException there is a problem using the supplied configuration.
      */
-    public SuccessMessageResponse activateServerListWithStoredConfig(String       userId,
-                                                                     List<String> serverNames)
-    {
-        String                 startUpMessage = null;
-        SuccessMessageResponse response       = new SuccessMessageResponse();
+    public SuccessMessageResponse activateServerListWithStoredConfig(String userId,
+                                                                     List<String> serverNames) {
+        String startUpMessage = null;
+        SuccessMessageResponse response = new SuccessMessageResponse();
 
         response.setRelatedHTTPCode(200);
 
-        if (serverNames != null)
-        {
-            for (String serverName : serverNames)
-            {
-                if (serverName != null)
-                {
+        if (serverNames != null) {
```

Comment:
There are _many_ formatting changes to the code in this one -- and while I personally prefer the new style, I thought per Dev Guidelines we were meant to stick to the original style of the code we change (and I recall on one of our calls @mandy-chessell suggesting she was spending time fixing formatting again after-the-fact)?

Reply:
@mandy-chessell  @cmgrote I will fix the formatting

Reply:
fixed formatting